src/app/api/auth/login/index.ts

```diff
@@ -3,22 +3,30 @@ import baseUrl from "@/app/utils/baseUrl";
 import Swal from "sweetalert2";
 import setCookies from "./setToken";
 
-const login = async (data:any) => {
+const login = async ( data: any ) => {
```

Comment:
I think that you can create an interface or type to add to set to the environment variable. This for you to know which type is sent in the endpoint.